pkg/scheduler/api/resource_info.go

```diff
@@ -187,17 +187,38 @@ func (r *Resource) Add(rr *Resource) *Resource {
 	return r
 }
 
+// AddBaseLeft is used to add two given resources based on the resource list of Resource r
+func (r *Resource) AddBaseLeft(rr *Resource) *Resource {
```

Comment:
What's the different between `Add` and `AddBaseLeft`?

Reply:
@huone1

Reply:
if a resource is not in r but in rr,  AddBaseLeft function will skip the  add operation of the resource; Add function will set the resource in r to zero and add them。

Reply:
Got you point. The logic is a little trick. Let's think whether there is a better idea for this.
AddbaseLeft function
input:
r cpu 1
rr cpu1 gpu1
output:
r cpu: 2

Add function
input:
r: cpu 1
rr: cpu1 gpu1
output:
r cpu: 2, gpu: 1